dashboard/Home.py

```diff
@@ -4,5 +4,43 @@
 
 if __name__ == "__main__":
 
-    st.title("Main Page")
-    st.title("Some intro stufff wooo")
+    st.set_page_config(
+        page_title="Media Polarisation Dashboard",
+        page_icon="📰",
+        layout="wide",
+        initial_sidebar_state="expanded"
+    )
+
+    st.title("Welcome to the Media Polarisation Dashboard")
+    st.markdown("""
+    ## Understanding Polarised Media Coverage and Sentiment
+                
+    In today's highly polarised media landscape, news outlets often present different
```

Comment:
Strong intro and explanation of why this dashboard is useful - this is going to be very useful!